README.adoc

```diff
@@ -0,0 +1,94 @@
+= C++ example project scanned on SonarCloud using Travis CI
+// URIs:
+:uri-qg-status: https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-autotools-gcov-travis-sc
+:img-qg-status: https://sonarcloud.io/api/project_badges/measure?project=sonarsource-cfamily-examples_linux-autotools-gcov-travis-sc&metric=alert_status
+:uri-build-status: https://app.travis-ci.com/sonarsource-cfamily-examples/linux-autotools-gcov-travis-sc
+:img-build-status: https://app.travis-ci.com/sonarsource-cfamily-examples/linux-autotools-gcov-travis-sc.svg?branch=main
+
+image:{img-build-status}[Build Status, link={uri-build-status}]
+image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
+
+*This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-autotools-gcov-travis-sc[SonarCloud]!*
+
+:note-caption: :information_source:
+NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
+
+It is very easy to analyze a C, C++ and Objective-C project with SonarCloud https://docs.travis-ci.com/user/sonarcloud/[on Travis CI]:
+
+. Create a `sonar-project.properties` file to store your configuration
+. In your `.travis.yml` file:
+.. Activate the https://docs.travis-ci.com/user/sonarcloud/[Travis Add-on] and add it to your `.travis.yml` file:
+* Specify your GitHub `organization`
+* add your encrypted `SONAR_TOKEN`
+.. Wrap your compilation with the https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper[Build Wrapper]
```

Comment:
What about adding a step regarding running tests and coverage?

Reply:
good point. I added the steps